src/ebmc/ebmc_parse_options.cpp

```diff
@@ -65,20 +67,18 @@ int ebmc_parse_optionst::doit()
   {
     if(cmdline.isset("diatest"))
     {
-      throw ebmc_errort() << "This option is currently disabled";
-
-#if 0
       if(!cmdline.isset("statebits"))
-      {
-        error("error: must provide number of state bits");
-        help();
-        return 1;
-      }
+        throw ebmc_errort() << "must provide number of state bits";
 
-      diatest(bound, atoi(cmdline.getval("statebits")));
+      if(!cmdline.isset("bound"))
+        throw ebmc_errort() << "must provide bound";
+
+      auto statebits = unsafe_string2size_t(cmdline.get_value("statebits"));
+      auto bound = unsafe_string2size_t(cmdline.get_value("bound"));
```

Comment:
I believe these should use `string2optional_size_t` for user-provided input is being parsed. If either returns `nullopt`, an exception can be thrown.

Reply:
done